javascript-light-bulb/main.js

```diff
@@ -1,9 +1,8 @@
 var $container = document.querySelector('.container');
 var $lightSwitch = document.querySelector('.light-switch');
-var count = 0;
+
 function lightSwitchFlicks() {
-  count++;
-  if (count % 2 !== 0) {
+  if ($container.className === 'container off') {
```

Comment:
Now your logic is dependent on the state of the DOM to know what to do, which is less than ideal. You should follow my initial suggestion of using a variable containing a boolean value to keep track of if the light is currently on or not, and use that variable's value to decide what action to take when the lightbulb is clicked.